lms/djangoapps/courseware/tests/test_i18n.py

```diff
@@ -96,35 +141,44 @@ def test_es419_acceptance(self):
     def test_unreleased_lang_resolution(self):
         # Regression test; LOC-85
         self.release_languages('fa')
+        self.user_login()
```

Comment:
Nit: move the `user_login` to this class' `setUp` method since it's required by all test methods in this class.